README.rst

```diff
@@ -93,15 +129,76 @@ and define some required methods:
         def get_description(self):
             return self.description
 
+        @transaction.atomic
+        @transition(
+            field=status,
+            source=[order_status.PENDING],
+            target=order_status.PROCESSING,
+        )
+        def set_as_paid(self):
+            self.is_paid = True
+            self.paid_date = timezone.now()
+
+        @transaction.atomic
+        @transition(
+            field=status,
+            source=[
+                order_status.PENDING,
+                order_status.PROCESSING,
+                order_status.ON_HOLD,
+                order_status.FAILED,
+            ],
+            target=order_status.CANCELLED,
+        )
+        def set_as_cancelled(self):
+            pass
+
+        @transaction.atomic
+        @transition(
+            field=status, source=order_status.PROCESSING, target=order_status.COMPLETED
+        )
+        def set_as_completed(self):
+            pass
+
 .. note:: If you already have an Order model and don't want to subclass ``AbstractOrder``
     just make sure you implement all methods.
 
-Inform getpaid of your Order model in ``settings.py`` and provide settings for payment backends:
+Define a ``Payment`` model by subclassing ``getpaid.models.AbstractPayment``
+and define some required methods:
 
 .. code-block:: python
 
+    # payments/models.py
+    from getpaid.models import AbstractPayment
+
+    class MyCustomPayment(AbstractPayment):
+
+        def __str__(self):
+            return f"Payment #{self.id}"
+
+        def on_mark_as_paid(self, **kwargs):
+            self.order.set_as_paid()
+            self.order.save()
+
+.. note:: If you already have an Order model and don't want to subclass ``AbstractOrder``
+    just make sure you implement all methods.
+
+Inform getpaid of your Order & Payment model in ``settings.py`` and provide settings for payment backends:
+
+.. code-block:: python
+
+    FRONTEND_URL = "https://mydomain.tld"
+    assert not FRONTEND_URL.endswith("/")
```

Comment:
I wonder if that is safe to apply on all projects. Maybe we should just use urljoin and don't care about slash?